lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -144,6 +144,17 @@ describe('CategoryRenderer', () => {
 
     const audits = categoryDOM.querySelectorAll('.lh-audit');
     assert.equal(audits.length, category.auditRefs.length, 'renders correct number of audits');
+
+    assert.equal(categoryDOM.querySelector('.lh-gauge__wrapper--plugin'), null);
+  });
+
+  it('plugin category has plugin badge', () => {
+    const category = JSON.parse(
+      JSON.stringify(sampleResults.reportCategories.find(c => c.id === 'seo')));
+    // Any category not of the core
+    category.id = 'lighthouse-plugin-someplugin';
+    const categoryDOM = renderer.render(category, sampleResults.categoryGroups);
+    assert.ok(categoryDOM.querySelector('.lh-gauge__wrapper--plugin'));
```

Comment:
maybe assert slightly more...e.g. there's only one plugin wrapper in the `categoryDOM`, and `pluginWrapper.querySelector('.lh-gauge__label').textContent` is the category's `title`

Reply:
There's only one category gauge being rendered here, so there's could only be the one wrapper to assert on. I think your advice makes sense only if this test was rendering the whole report?

Reply:
> There's only one category gauge being rendered here, so there's could only be the one wrapper to assert on. I think your advice makes sense only if this test was rendering the whole report?

oh yeah :)

- the `.lh-gauge__label` check still makes sense, though, and that reminds me
- we should have another test similar to this but out in `report-renderer-test.js`, since report-renderer is in charge of rendering the `lh-scores-header` with all the gauges in it.

Reply:
done